packages/server/src/registration/verifyRegistrationResponse.ts

```diff
@@ -289,6 +295,8 @@ export type VerifiedRegistrationResponse = {
     userVerified: boolean;
     credentialDeviceType: CredentialDeviceType;
     credentialBackedUp: boolean;
+    origin: string;
+    rpId?: string;
```

Comment:
Same thing here, for consistency's sake:

```suggestion
    rpID?: string;
```